generator/src/test/java/com/scottlogic/deg/generator/generation/TestGenerationConfigSource.java

```diff
@@ -71,4 +72,9 @@ public boolean isEnableTracing() {
     public File getProfileFile() {
         throw new UnsupportedOperationException();
     }
+
+    @Override
+    public boolean shouldViolate() {
+        return violate;
```

Comment:
return `false` here, no need to hold the field `violate` either, at this stage anyhow